projects/nx-verdaccio/src/plugin/targets/create-targets.ts

```diff
@@ -22,20 +23,31 @@ export function createTargets(
     return {};
   }
 
+  // unfortunately namedInputs are not picked up by tasks graph: Error: Input 'build-artifacts' is not defined
```

Comment:
When you pass your own `namedInputs` (like you would in a project.json file) via the inferred tasks plugin, the tasks pipeline ignores them and throws this error.
Some Nx plugins use the `default` `namedInput`, probably for that reason, but I'm concerned that if developers change those inputs, it might lead to undesired behavior.

I still need to figure out why those namedInputs must be statically defined, something related to the task runner maybe?
I will ask on Discord.